MapboxNavigation/MGLStyle.swift

```diff
@@ -25,55 +25,31 @@ extension MGLStyle {
      */
     public class var navigationNightStyleURL: URL {
         get {
-            if(MGLAccountManager.hasChinaBaseURL){
+            if MGLAccountManager.hasChinaBaseURL {
                 return mapboxChinaDayStyleURL
             }
             return URL(string:"mapbox://styles/mapbox-map-design/ckd6dnz2q0m0q1io1mssumxqd")!
         }
     }
     
     /**
-     Returns the URL to the given version of the navigation guidance style. Available version are 1, 2, 3, and 4.
+     Returns the URL to the given version of the Mapbox Navigation Day style. Available versions are 1, 2, 3, and 4.
      
-     We only have one version of navigation guidance style in China, so if you switch your endpoint to .cn, it will return the default day style.
+     We only have one version of Mapbox Navigation Day style in China, so if you switch your endpoint to .cn, it will return the default day style.
      */
-    public class func navigationGuidanceDayStyleURL(version: Int) -> URL {
-        if(MGLAccountManager.hasChinaBaseURL){
-            return mapboxChinaDayStyleURL
-        }
-        return URL(string:"mapbox://styles/mapbox/navigation-guidance-day-v\(version)")!
-    }
-    
-    /**
-     Returns the URL to the given version of the navigation guidance style. Available version are 2, 3, and 4.
-     
-     We only have one version of navigation guidance style in China, so if you switch your endpoint to .cn, it will return the default night style.
-     */
-    public class func navigationGuidanceNightStyleURL(version: Int) -> URL {
-        if(MGLAccountManager.hasChinaBaseURL){
-            return mapboxChinaNightStyleURL
-        }
-        return URL(string: "mapbox://styles/mapbox/navigation-guidance-night-v\(version)")!
-    }
-    
-    /**
-     Returns the URL to the given version of the Mapbox Navigation Preview Day style. Available versions are 1, 2, 3, and 4.
-     
-     We only have one version of Navigation Preview style in China, so if you switch your endpoint to .cn, it will return the default day style.
-     */
-    public class func navigationPreviewDayStyleURL(version: Int) -> URL {
+    public class func navigationDayStyleURL(version: Int) -> URL {
         if MGLAccountManager.hasChinaBaseURL {
             return mapboxChinaDayStyleURL
         }
         return URL(string:"mapbox://styles/mapbox/navigation-guidance-day-v\(version)")!
```

Comment:
Even though methods were renamed URLs were left intact (still contain `guidance`). If there are updated URLs for versions prior to v5 please let me know. I've done a brief search and do not see anything.

Reply:
The v5 style probably won‘t have “guidance“ in the URL.

Reply:
Eventually I wasn't able to find anything for versions 4 and earlier, so added versions range check.